pom.xml

```diff
@@ -598,13 +598,42 @@
           <stagingProgressTimeoutMinutes>15</stagingProgressTimeoutMinutes>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>set number of slashes in URL</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <exportAntProperties>true</exportAntProperties>
+              <target>
+                <condition property="urlSlashesOsModifier" value="//">
+                  <and>
+                    <not>
+                      <os family="windows"></os>
+                    </not>
+                  </and>
+                </condition>
+                <condition property="urlSlashesOsModifier" value="///">
+                  <and>
+                    <os family="windows"></os>
```

Comment:
ow windows.... gotta love that.

Funny enough, I'm on windows 11, but all work is done under WSL, so never noticed the problem